### DISCUSSION TITLE
Can we use Plain RDS PostgreSQL (non-Aurora) and IAM authentication?

### DISCUSSION THREAD

**Discussion by Captain1653** [aws/aws-advanced-jdbc-wrapper#1043] [Q&A] [answered]
Hi,

We want to use **Plain RDS PostgreSQL (non-Aurora)** and IAM authentication. Is it possible?  I've found [this issue](https://github.com/aws/aws-advanced-jdbc-wrapper/issues/1034), but there are no any answers or docs updates.

I've found some related discussions, but I'm not sure:
* [one](https://github.com/aws/aws-advanced-jdbc-wrapper/discussions/477)
* [two](https://github.com/aws/aws-advanced-jdbc-wrapper/discussions/216)
* [three](https://github.com/aws/aws-advanced-jdbc-wrapper/discussions/710)

Kind regards

**Comment by davecramer** [comment, marked as answer]
@Captain1653 it should work fine. Please let us know if it does not.

**Reply by Captain1653** [reply]
Thank you for the quick reply. 

We will try and then let you know.

**Comment by hsuamz** [comment]
@Captain1653 , for visibility we recently updated our documentation #1034 to better reflect that RDS PostgreSQL supports IAM with the driver.

If you experience any issues, please reach out and we'll be happy to help.
